docs/source/usage/plugins/xrayScattering.rst

```diff
@@ -0,0 +1,127 @@
+
+.. math::
+
+   \Phi({\bf q}) &= \frac{r_e}{d}  \int_{t} \mathrm{d}t \int_{V} \mathrm{d}V \phi({\bf r}, t) n({\bf r}, t) \\
+   I &= \left| \Phi \right|^2
+
+
+============================== ================================================================================
+Variable                       Meaning
+============================== ================================================================================
+:math:`\Phi`                   Scattered amplitude
+:math:`\bf q`                  Scattering vector with :math:`|{\bf q}| = \frac{4 \pi \sin \theta}{\lambda}`
+:math:`\theta`                 Scattering angle. :math:`2\theta` is the angle between the incoming and the scattered k-vectors.
+:math:`\lambda`                Probing beam wavelength
+:math:`n`                      Electron density
+:math:`\phi`                   Incoming wave amplitude
+:math:`I`                      Scattering intensity
+:math:`d`                      Screen distance
+:math:`r_e`                    classical electron radius
+
+============================== ================================================================================
+
+
+For the free electrons, the density :math:`n` is just their number density, for ions it is the bound electrons density of the species.
+This plugin will automatically switch to bound electrons density for species having the `boundElectrons` property.
+
+The volume integral is realized by a discrete sum over the simulation cells and the temporal integration reduces to accumulating the amplitude over simulation time steps.
+
+.. note::
+    This calculation is based on the kinematic model of scattering. Multiple Scattering CAN NOT be handled in this model.
+
+.param file
+^^^^^^^^^^^
+
+The `xrayScattering.param` file sets the x-ray beam alignment as well as its temporal and transverse envelope.
+
+.. note::
+    At the moment the translation (to the side center + offset) is not working correctly.
+    For that reason, the envelopes and the offset can't be set in the ``.param`` file yet.
+    The probe is always a plane wave.
+    Beam rotation works.
+
+The alignment settings define a beam coordinate system with :math:`\hat{z}  = \hat{k}` and :math:`\hat{x}`, :math:`\hat{y}` perpendicular to the x-ray propagation direction.
+It is always a right-hand system. It is oriented in such way that for propagation parallel to the PIC x- or y-axis (`Side`: `X`, `XR`, `Y` or `YR`) :math:`\hat{x}_{\text{beam}} = - \hat{z}_{\text{PIC}}` holds and if :math:`{\bf k }` is parallel to  the PIC z-axis (`Side`: `Z` or `ZR`),  :math:`\hat{x}_{\text{beam}} = - \hat{y}_{\text{PIC}}` holds.
+The orientation can be then fine adjusted with the `RotationParam` setting.
+.. TODO: Figures showing the beam coordinate system orientation in the PIC system.
+
+.. TODO: Add other parameters after the coordinate transform has been fixed and the settings have been moved back to the .param file.
```

Comment:
So the coordinate transform fix is rather a thing for the next PR. The first TODO, I could do it before we merge but it can be added in an another PR as well.

Reply:
Thanks for explanations.  For the first TODO I think it's fine to add both as a follow-up PR or to this one as it's just documentation and does not change the logic, so does not really influence test results or review points.